lib/features/identity_creator/presentation/identity_creator_controller.dart

```diff
@@ -568,16 +595,7 @@ class IdentityCreatorController extends BaseController {
         logError("IdentityCreatorController::_insertInlineImage: context is unmounted");
       }
     } else {
-      if (PlatformInfo.isWeb) {
-        richTextWebController?.insertImageAsBase64(platformFile: file, maxWidth: maxWidth);
-      } else if (PlatformInfo.isMobile) {
-        richTextMobileTabletController?.insertImageData(platformFile: file, maxWidth: maxWidth);
-        if (file.path != null) {
-          _deleteCompressedFileOnMobile(file.path!);
-        }
-      } else {
-        logError("IdentityCreatorController::_insertInlineImage: Platform not supported");
-      }
+      publicAssetController?.uploadFileToBlob(file);
```

Comment:
What about if `publicAsset` not supported?

Reply:
@hoangdat should we keep using base64 if publicAsset capability is not supported?

Reply:
keep base64 for now. Thanks

Reply:
Done